tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/BulkheadFutureTest.java

```diff
@@ -77,126 +82,92 @@ public void beforeTest(final ITestContext testContext) {
      * Tests that the Future that is returned from an asynchronous bulkhead
      * method can be queried for Done OK before and after a goodpath .get()
      */
-    @Test()
+    @Test
     public void testBulkheadMethodAsynchFutureDoneAfterGet() {
-
         Checker fc = new FutureChecker(SHORT_TIME);
         Future<String> result = null;
 
         try {
             result = bhBeanMethodAsynchronousDefault.test(fc);
         }
         catch (InterruptedException e1) {
-            Assert.fail("Unexpected interruption", e1);
+            fail("Unexpected interruption", e1);
         }
 
-        Assert.assertFalse(result.isDone(), "Future reporting Done when not");
+        assertFalse(result.isDone(), "Future reporting Done when not");
         try {
-            String r;
-            Assert.assertTrue("GET.".equals(r = result.get()), r);
-            Assert.assertTrue("GET.GET_TO.".equals(r = result.get(1, TimeUnit.SECONDS)), r);
+            assertEquals(result.get(), "GET.");
+            assertEquals(result.get(1, TimeUnit.SECONDS), "GET.GET_TO.");
         }
         catch (Throwable t) {
-            Assert.assertNull(t);
+            fail("Unexpected exception", t);
         }
-        Assert.assertTrue(result.isDone(), "Future done not reporting true");
+        assertTrue(result.isDone(), "Future done not reporting true");
     }
 
     /**
      * Tests that the Future that is returned from a asynchronous bulkhead
      * method can be queried for Done OK even if the user never calls get() to
      * drive the backend (i.e. the method is called non-lazily)
      */
-    @Test()
+    @Test
     public void testBulkheadMethodAsynchFutureDoneWithoutGet() {
-
         Checker fc = new FutureChecker(SHORT_TIME);
-        Future<String> result = null;
         try {
-            result = bhBeanMethodAsynchronousDefault.test(fc);
+            final Future<String> result = bhBeanMethodAsynchronousDefault.test(fc);
+            assertFalse(result.isDone(), "Future reporting Done when not");
+            await().atMost(SHORT_TIME * SHORT_TIME, MILLISECONDS).untilAsserted(()-> assertTrue(result.isDone()));
```

Comment:
You've changed an effective wait of `SHORT_TIME * 100` to `SHORT_TIME * SHORT_TIME`

`SHORT_TIME` is actually 100, but it looks weird in the code to just be squaring the timeout time.

Reply:
done

Reply:
done